### PR TITLE
Address issue #429

### PR DESCRIPTION
**Pull request by melcorr**
Commit 1 adds web copy. Image coming soon. 
Commit 2 updates image, adds desktop and updates content. 

https://melcorr.github.io/pulpproject.org/

### CODE REVIEW

**Comment by melcorr** [comment]
Let me know what you think @daviddavis

**Review by fao89 — APPROVED** [review]
❤️

**Comment by daviddavis** [comment]
This looks great!

File: index.md [hunk 1]
```diff
@@ -12,17 +12,29 @@ homepage: true
...
+If you have to manage hundreds or thousands of packages, Pulp can help!
+
+![](/images/pulp-workflow-architecture-ha/pulp-101.png)
+
+
+* **Ensure stability and continuity:** External content sources can go offline unexpectedly.
+If you want to ensure that you always have what you need, Pulp can help.
+* **Stop using rsync:** Pulp is designed with complex content management workflows and disk optimization in mind.
+If your sync script is letting you down, Pulp can help.
+* **Reduce rate limiting:** From one day to the next, third party platforms can introduce rate limiting and change the conditions of service.
```
Comment (daviddavis) [inline index.md]:
```suggestion
* **Reduce rate limiting:** From one day to the next, third-party platforms can introduce rate limiting and change the conditions of service.
```

File: index.md [hunk 2]
```diff
@@ -12,17 +12,29 @@ homepage: true
...
+If you want to reduce operation costs by having your team consume content from Pulp rather than third parties, Pulp can help.
+* **Distribute content privately:** Sometimes you need a way to distribute private content you have developed in house.
+If you want to keep your private packages off third-party platforms and distribute them with ease, Pulp can help.
```
Comment (daviddavis) [inline index.md]:
```suggestion
If you want to keep your private packages off third-party platforms and distribute them internally with ease, Pulp can help.
```

File: index.md [hunk 3]
```diff
@@ -12,17 +12,29 @@ homepage: true
...
+* **Experiment without risk:** Every change to content hosted in Pulp creates a new repository version.
+If you need to pin packages to certain versions to ensure stability and repeatability, Pulp can help.
```
Comment (daviddavis) [inline index.md:29]:
```suggestion
If you need to pin packages to certain versions to ensure stability or rollback changes to repos, Pulp can help.
```

File: index.md [hunk 4]
```diff
@@ -12,17 +12,29 @@ homepage: true
...
+... And much more!
+
 
-Pulp has a [REST API](https://docs.pulpproject.org/rest_api.html) for management.
 
 Pulp is free and open-source, and we invite you to [join us on GitHub](https://github.com/pulp/).
 
-Follow us on [Twitter](https://twitter.com/pulpproj) or check out our [Youtube
-channel](https://www.youtube.com/PulpProject).
+Come chat to us on our [community forum](https://discourse.pulpproject.org/).
```
Comment (daviddavis) [inline index.md]: I might add matrix info here too?

File: index.md (hunk 3 above)
Comment (daviddavis) [reply index.md:29]: Maybe this change doesn't make sense. I was just thinking it'd be nice to call out the ability to rollback changes to repo somehow.

**Review by daviddavis — APPROVED** [review]
Some small suggestions from me. Thanks for doing this!

File: index.md (hunk 3 above)
Comment (melcorr) [reply index.md:29]: I committed it and now see that it reads badly. I love the term repeatability. I'll try weave in rollback.

File: index.md (hunk 4 above)
Comment (melcorr) [reply index.md]: Oh yes!

File: index.md (hunk 3 above)
Comment (melcorr) [reply index.md:29]: Would this work?

**Comment by melcorr** [comment]
If there aren't any further comments, I'll merge @daviddavis ?

**Comment by daviddavis** [comment]
:shipit:

**Pull request merged**